components/chainSelect/ChainSelect.tsx

```diff
@@ -61,20 +61,28 @@ const ChainSelect = () => {
   const [tempRegistryName, setRegistryName] = useState(state.chain.registryName);
   const [tempExplorerLink, setExplorerLink] = useState(state.chain.explorerLink);
 
-  const url = "https://api.github.com/repos/cosmos/chain-registry/contents";
+  const chainsUrl = "https://api.github.com/repos/cosmos/chain-registry/contents";
+  const testnetsUrl = "https://api.github.com/repos/cosmos/chain-registry/contents/testnets";
 
   const getGhJson = useCallback(async () => {
     // getting chain info from this repo: https://github.com/cosmos/chain-registry
     try {
-      const res = await axios.get(url);
-      const chains = res.data.filter((item: GithubChainRegistryItem) => {
-        return item.type == "dir" && !item.name.startsWith(".") && item.name != "testnets";
-      });
-      setChainArray(chains);
-      const options = chains.map(({ name }: GithubChainRegistryItem, index: number) => {
-        return { label: name, value: index };
-      });
+      const { data: chains } = await axios.get(chainsUrl);
+      const { data: testnets } = await axios.get(testnetsUrl);
```

Comment:
Let's download them in parallel with `Promise.all([axios.get(chainsUrl), axios.get(testnetsUrl)])`